docs/src/user/manual/explore/layers.md

```diff
@@ -95,6 +96,8 @@ Welcome to the documentation for map layers available in the application. This d
 
     2. **Property Boundaries Display**: White lines on the map indicate the property boundaries and land divisions.
 
+    3. **Parent farm Cname:** Click on the map to view the parent farm's Cname.
```

Comment:
```suggestion
    3. **Cadastral Cname:** Click on the map to view a cadastral's Cname. Depending on the zoom level this will be the Parent Farm, Farm Portion, Holding, or Erf.
```

![image](https://github.com/kartoza/sawps/assets/100694811/11d44df6-6bfb-48c4-95a8-da2e14131f78)